drivers/soc/bl702/std/src/bl702_glb.c

```diff
@@ -150,6 +150,7 @@ GLB_ROOT_CLK_Type ATTR_CLOCK_SECTION GLB_Get_Root_CLK_Sel(void)
  * @return SUCCESS or ERROR
  *
 *******************************************************************************/
+#ifndef BFLB_USE_ROM_DRIVER
 BL_Err_Type ATTR_CLOCK_SECTION GLB_Set_System_CLK_Div(uint8_t hclkDiv, uint8_t bclkDiv)
```

Comment:
this api cannot use romdirver

Reply:
But it works in presented implementation and in legacy SDK (v1.4.5). Is the bootrom API implementation is totally broken?
I can't find any errata regarding boot rom code. Do you have such document?

Reply:
not provide.

Reply:
and this api has disabled in romapi.c

Reply:
I can't understand such canceling of RomAPI without any reasonable argument, f.e. errata or other similar documentation from manufacturer. Especially if it works properly on different boards and in different conditions.  :man_facepalming: